<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResponderIncidenciaForm</class>
 <widget class="QWidget" name="ResponderIncidenciaForm">
  <property name="windowTitle">
   <string>Responder Incidencia</string>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>350</height>
   </size>
  </property>
  <property name="styleSheet">
   <string>
QWidget {
    background-color: #e6f2ff;
    font-family: Arial, sans-serif;
    font-size: 15px;
    color: #1a1a1a;
}
QLabel#tituloFijoLabel, QLabel#descripcionFijoLabel {
    color: #0080b2;
    font-size: 16px;
    font-weight: bold;
    margin-top: 8px;
}
QLabel#tituloLabel, QLabel#descripcionLabel {
    color: #1a1a1a;
    font-size: 15px;
    background: #f7fbff;
    border: 1px solid #d0e3f2;
    border-radius: 8px;
    padding: 8px;
    margin-bottom: 6px;
}
QTextEdit#respuestaBox {
    border: 1.5px solid #80bfff;
    border-radius: 8px;
    padding: 8px;
    background-color: #f7fbff;
    font-size: 15px;
    color: #222;
    min-height: 90px;
}
QPushButton#enviarBtn {
    background-color: #00cc99;
    color: white;
    border-radius: 14px;
    padding: 10px 0;
    font-size: 16px;
    margin-top: 10px;
    margin-bottom: 4px;
}
QPushButton#enviarBtn:hover {
    background-color: #009973;
}
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>16</number>
   </property>
   <property name="bottomMargin">
    <number>16</number>
   </property>
   <item>
    <widget class="QLabel" name="tituloFijoLabel">
     <property name="text">
      <string>Título:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="tituloLabel">
     <property name="text">
      <string>(Aquí se mostrará el título de la incidencia)</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="descripcionFijoLabel">
     <property name="text">
      <string>Descripción:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="descripcionLabel">
     <property name="text">
      <string>(Aquí se mostrará la descripción de la incidencia)</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="respuestaBox">
     <property name="placeholderText">
      <string>Escribe aquí la respuesta del administrador...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="enviarBtn">
     <property name="text">
      <string>Enviar respuesta</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>